pennylane/beta/interfaces/autograd.py

```diff
@@ -116,6 +116,8 @@ def get_parameters(self, trainable_only=True):  # pylint: disable=missing-functi
                 for idx, p in enumerate(self._all_parameter_values)
                 if idx in self.trainable_params
             ]
+        else:
+            params = self._all_parameter_values

```

Comment:
I don't think this is needed any more, given line 111

Reply:
Good point, removed!